Fix formatting in a couple docs (#47523)
CleverRaven/Cataclysm-DDA: doc/SOUNDPACKS.md

# Soundpacks

> A soundpack can be installed in the `data/sound` directory. It has to be a subdirectory that contains at least a file named `soundpack.txt`. It can include any number of json files which add any number of `sound_effect` or playlist.

Reviewer: ```suggestion
A soundpack can be installed in the `data/sound` directory. It has to be a subdirectory that contains at least a file named `soundpack.txt`. It can include any number of json files which add any number of `sound_effect` or playlists.
```

> 

Reviewer: I suggest running JSON examples through the [CDDA JSON formatter](https://dev.narc.ro/cataclysm/format.html). They may require putting some extra lines to make them a complete file, however.

Author: Done. Neat tool! It did produce one somewhat long line on [L72](https://github.com/CleverRaven/Cataclysm-DDA/pull/47523/files#diff-ae5ecf47213a1cc0a5651fe6f88f494a4f162e30e432c41139555a5fb31fbf77R72), but that's more of a nitpick.

```javascript
[
  {
    "type": "sound_effect",

>     "id" : "menu_move",
>     "volume" : 100,
>     "files" : [

Reviewer: ```suggestion
    "id": "menu_move",
    "volume": 100,
    "files": [
```
JSON files are required to have a specific format.

      "nenadsimic_menu_selection_click.wav"
    ]
  },
  {
    "type": "sound_effect",

>     "id" : "fire_gun",
>     "volume" : 90,
>     "variant" : "bio_laser_gun",
>     "files" : [

Reviewer: ```suggestion
    "id": "fire_gun",
    "volume": 90,
    "variant": "bio_laser_gun",
    "files": [
```

```javascript
[
  {
    "type": "sound_effect",
    "id": "menu_move",
    "volume": 100,
    "files": [ "nenadsimic_menu_selection_click.wav" ]
  },
  {
    "type": "sound_effect",
    "id": "fire_gun",
    "volume": 90,
    "variant": "bio_laser_gun",
    "files": [ "guns/energy_generic/weapon_fire_laser.ogg" ]
  }
]
```

Adding variety: If for a certain `id`'s `variant` multiple `files` are defined, they will be chosen at random when `variant` is played.

The volume key may range from 0-100.

> Cataclysm has its own set of user-controllable volumes that will additionally affect the sound. These range from 0-128, and the default is 100. This means that at default volume, any sound that Cataclysm plays will default to playing at about 78% of the maximum; if you are working on sounds in an external audio editor, expect Cataclysm at default volume settings to play that sound file back more-quietly than your editor does.

Reviewer: ```suggestion
Cataclysm has its own set of user-controllable volumes that will additionally affect the sound. These range from 0-128, and the default is 100. This means that at default volume, any sound that Cataclysm plays will default to playing at about 78% of the maximum; if you are working on sounds in an external audio editor, expect Cataclysm at default volume settings to play that sound file back quieter than your editor does.
```
Or "more quietly", without the hyphen.

* `melee_hit_metal   chainsaw_on`
* `weapon_theme      chainsaw`

### Monster death and bite attacks

* `mon_death zombie_death|zombie_gibbed`
* `mon_bite bite_miss|bite_hit`
* `melee_attack monster_melee_hit`
* `player_laugh laugh_f|laugh_m`

### Player movement sfx

Important: `plmove <terrain>` has priority over default `plmove|walk_<what>` (excluding `|barefoot`).

Example: if `plmove|t_grass_long` is defined it will be played before default `plmove|walk_grass` default for all grassy terrains.

* `plmove <terrain>|<vehicle_part>`
* `plmove walk_grass|walk_dirt|walk_metal|walk_water|walk_tarmac|walk_barefoot|clear_obstacle`

### Fatigue

* `plmove fatigue_m_low|fatigue_m_med|fatigue_m_high|fatigue_f_low|fatigue_f_med|fatigue_f_high`

### Player hurt

* `deal_damage hurt_f|hurt_m`

### Player death and end-game

* `clean_up_at_end game_over|death_m|death_f`

### Various bionics

* `bionic elec_discharge|elec_crackle_low|elec_crackle_med|elec_crackle_high|elec_blast|elec_blast_muffled|acid_discharge|pixelated`
* `bionic bio_resonator|bio_hydraulics|`

> ### Various tools/traps

Reviewer: ```suggestion
### Various tools/traps being used
```
